packages/atomic/src/components/common/generated-answer/styles/generated-answer.pcss

```diff
@@ -68,13 +73,51 @@
     @apply ml-auto;
   }
 
-  @container (max-width: 600px) {
+  @container (max-width: 37.5rem) {
     @mixin mobile-footer-layout;
   }
   @screen mobile-only {
     @mixin mobile-footer-layout;
   }
-  @container (max-width: 400px) {
+  @container (max-width: 25rem) {
     @mixin mobile-footer-xs-layout;
   }
 }
+
+[part='generated-answer-footer'] {
+  @container (max-width: 600px) {
```

Comment:
Is there a reason why this dimension is still in pixels?

Reply:
good catch! forgot to change this one, thanks!!